### ISSUE TITLE
fping rta in ms [sf#921467]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#315]
Submitted by mattkent on 2004-03-23 01:10:15

fping's help claims to take the critical and warn rta
values in milliseconds like ping, but as we can see it
doesn't. 
# ./check_fping -H pdc-pv1.pv -w 10,10% -c 100,25%

FPING OK - pdc-pv1.pv (loss=0%, rta=42.800000
ms)|loss=0%;10;25;0;100 rta=42799us;10000000;100000000;0
# ./check_ping -H pdc-pv1.pv -w 10,10% -c 100,25%

PING WARNING - Packet loss = 0%, RTA = 32.67 ms

As they always stay in 'OK' state, this sucks for
anyone like me who replaced the ping checks with fping
and didn't test the result  :)

Tiny patch against version in cvs fixes the inconsistency.
